### PR TITLE
Add cfitsio docs

### PR DESCRIPTION
**Pull request by Carldkennedy**

### CODE REVIEW

File: stanagepilot/software/libs/cfitsio.rst [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+Testing
+-------
+
+Fits file downloaded from `here <https://fits.gsfc.nasa.gov/samples/IUElwp25637mxlo.fits>`_.
+
+.. code-block ::
+
+        $ module load CFITSIO/3.49-GCCcore-10.2.0
+        $ module load GCC/GCCcore-10.2.0
+        $ gcc test.c -o test -lm /opt/apps/testapps/el7/software/staging/CFITSIO/3.49-GCCcore-10.2.0/lib/libcfitsio.so
```
Comment (jkwmoore) [inline stanagepilot/software/libs/cfitsio.rst:44]: Where does this test.c file come from / what is it doing?

Comment (jkwmoore) [reply stanagepilot/software/libs/cfitsio.rst:44]: Should we use the given multi-threaded example/sample programs? https://heasarc.gsfc.nasa.gov/FTP/software/fitsio/c/threadtest.tar.gz

Comment (Carldkennedy) [reply stanagepilot/software/libs/cfitsio.rst:44]: See https://jigthfxpfv3.atlassian.net/browse/STN-321

**Review by willfurnass — APPROVED** [review]
LGTM.  Do we defintely want to provide a `foss-2018b` -compatible version?

**Comment by Carldkennedy** [comment]
> LGTM. Do we defintely want to provide a `foss-2018b` -compatible version?

CFITSIO/3.45-GCCcore-7.3.0 has no dependant modules - will remove. Thank you @willfurnass

**Review by nikishe — APPROVED** [review]

**Review by nikishe — APPROVED** [review]

File: stanagepilot/software/libs/cfitsio.rst (hunk 1 above)
Comment (jkwmoore) [reply stanagepilot/software/libs/cfitsio.rst:68]: That's great but ideally the provenance of the tests is fully indicated on the docs for the software! Bit difficult for a new onlooker to decide A: what this is testing and B: whether this testing is sufficient proof.

**Pull request merged**